.cargo/config

```diff
@@ -13,7 +13,7 @@ rustdocflags = "--document-private-items"
 # things critical to correctness probably don't belong here.
 [target.x86_64-unknown-illumos]
 rustflags = [
-    "-C", "link-arg=-Wl,-znocompstrtab"
+    "-C", "link-arg=-Wl,-znocompstrtab,-R/usr/platform/oxide/lib/amd64"
```

Comment:
This is a workaround I was using to get around a missing run path.  The solution is likely somewhere in the `rpath` crate - I am diverting my attention to fixing this now, but if someone has pointers on how this should be done, that would be greatly appreciated.

Reply:
Spoke to dap about this and I plan to push a commit to clean up this mess.

Reply:
Note: opting to go with passing the oxide platform directory here after some discussion in chat.